spec/dummy_app/config.ru

```diff
@@ -2,5 +2,5 @@
 
 # This file is used by Rack-based servers to start the application.
 
-require ::File.expand_path("../config/environment", __FILE__)
+require ::File.expand_path("config/environment", __dir__)
```

Comment:
Rubocop reported this.  I'm not familiar with this but I'm assuming it's a safe change?

Reply:
Yes, that's alright.